File: node-ui/classes/Database/PoliticalGroups.js
const { DataTypes } = require('sequelize');
const sequelize = require('./base');

const PoliticalGroups = sequelize.define('political_groups', {
    name: {
        type: DataTypes.STRING(32),
        allowNull: false,
        primaryKey: true
    },
    country: {
        type: DataTypes.STRING(16),
        allowNull: false
    },
    logo_color: {
        type: DataTypes.STRING(32),
        allowNull: false
    },
    last_update: {
        type: DataTypes.DATEONLY,
        allowNull: false
    }
});

module.exports = PoliticalGroups

File: py-crawler/classes/databases/political_groups.py
from sqlalchemy import Column
from sqlalchemy import String, Date

from .base import Base

# political_groups table
# Author: Antonio Scardace

class PoliticalGroups(Base):

    __tablename__ = 'political_groups'

    name = Column('name', String(32), nullable=False, primary_key=True)
    country = Column('country', String(16), nullable=False)
    logo_color = Column('logo_color', String(32), nullable=False)
    last_update = Column('last_update', Date, nullable=False)

    def __init__(self, name: str):
        self.name = name